packages/tools/fluid-runner/src/test/sampleCodeLoaders/sampleCodeLoader.ts

```diff
@@ -24,7 +24,14 @@ export class SampleCodeLoader implements ICodeDetailsLoader {
 	}
 	public async load(): Promise<IFluidModuleWithDetails> {
 		return {
-			module: { fluidExport: new BaseContainerRuntimeFactory(new Map()) },
+			module: {
+				fluidExport: new BaseContainerRuntimeFactory({
+					registryEntries: new Map(),
+					initializeEntryPoint: async () => ({
```

Comment:
For all places that have this kind of entry point - What was the behavior before this change when someone requested "/" on the this container? It should have returned a 404 right? Maybe we should keep simiar behavior and simply return undefined?

Reply:
`initializeEntryPoint()` cannot return undefined as per the typing. This codeLoader is just used for our tests, so I think it's fine that it doesn't return anything useful.